config/nrfconnect/chip-module/Kconfig

```
@@ -254,4 +254,14 @@ config CHIP_DEVICE_ROTATING_DEVICE_UID
 	  A device rotating id unique id which will be generated if
 	  this config is not set in prj.conf file.
 
+config CHIP_DEVICE_ENABLE_KEY
+	string "A 128-bit value that triggers manufacturer-specific action."
```

```suggestion
	string "Enable Key for triggering test actions on device"
```
Actions are not necessarily manufacturer-specific. CSA can and will define them, too. Also, in general we don't need to use articles ("A something") in Kconfig prompts. Better to keep them shorter :)